crates/apollo-mcp-server/telemetry.toml

```diff
@@ -7,6 +7,7 @@ success = "Sucess flag indicator"
 raw_operation = "Graphql operation text and metadata used for Tool generation"
 client_name = "The client name that initializes with the MCP Server"
 client_version = "The client version that initializes with the MCP Server"
+session_id = "The value of the Mcp-Session-Id header in the HTTP request"
```

Comment:
Because the mcp-session-id header is present on both request and response, can we add trace attributes for both, so we can correlate end-to-end?